core/src/test/java/org/elasticsearch/search/sort/SortParserTests.java

```diff
@@ -47,7 +47,7 @@ public void testGeoDistanceSortParserManyPointsNoException() throws Exception {
         sortBuilder.endArray();
         sortBuilder.field("order", "desc");
         sortBuilder.field("unit", "km");
-        sortBuilder.field("sort_mode", "max");
+        sortBuilder.field("mode", "max");
```

Comment:
Yes, I also added a test tieh the deprecated "sort_mode" syntax to check that we throw an error in strict mode.